great_expectations/execution_engine/sparkdf_execution_engine.py

```diff
@@ -286,10 +287,10 @@ def get_batch_data_and_markers(
                     """
                 )
 
-        elif isinstance(batch_spec, PathBatchSpec):
+        elif isinstance(batch_spec, (PathBatchSpec, GlueDataCatalogBatchSpec)):
             reader_method: str = batch_spec.reader_method
             reader_options: dict = batch_spec.reader_options or {}
-            path: str = batch_spec.path
+            reader_arg: str = batch_spec.path
```

Comment:
@lccasagrande Question: how come it is now `reader_arg` as opposed to `path` (while in `batch_spec` it is still stored as `path`)?  If not critical, perhaps we could revert this change?  I will defer to you the best way to go.  Thank you!

Reply:
Done, I reverted the changes.